localstack/utils/aws/aws_stack.py

```diff
@@ -193,10 +194,17 @@ def get_boto3_region() -> str:
     return boto3.session.Session().region_name
 
 
-def is_internal_call_context(headers):
+def is_internal_call_context(headers) -> bool:
```

Comment:
This is fine for now, preferably we would migrate all this to the is_internal_call property of the RequestContext at some point. Out of scope of this PR of course.